Sources/SwiftParserDiagnostics/ParserDiagnosticMessages.swift

```diff
@@ -306,6 +306,14 @@ public struct IdentifierNotAllowedInOperatorName: ParserError {
   }
 }
 
+public struct InvalidFloatLiteralMissingLeadingZero: ParserError {
+  public let integerLiteral: TokenSyntax
```

Comment:
What do you think about naming this `decimalDigits`? `integerLiteral` is a bit confusing to me since this is an error about floating point literals.